tests/middleware/tests.py

```diff
@@ -763,6 +789,7 @@ def test_no_compress_incompressible_response(self):
         self.assertIsNone(r.get('Content-Encoding'))
 
 
+@ignore_warnings(category=RemovedInDjango20Warning)
```

Comment:
Seems like this test should be duplicated to test if the `ConditionalGetMiddleware` (with its new functionality) behaves correctly with `GZipMiddleware`?